app/src/main/java/de/danoeh/antennapod/ui/screen/playback/audio/AudioPlayerFragment.java

```diff
@@ -495,6 +496,11 @@ public boolean onMenuItemClick(MenuItem item) {
         if (itemId == R.id.disable_sleeptimer_item || itemId == R.id.set_sleeptimer_item) {
             new SleepTimerDialog().show(getChildFragmentManager(), "SleepTimerDialog");
             return true;
+        } else if (itemId == R.id.transcript_item) {
+            new TranscriptFragment()
+                    .initMedia(media)
```

Comment:
This will break when the Android system creates a new instance of the fragment. Instead, the arguments should be passed with `setArguments`. For most other fragments, we have a method `newInstance` that handles this. See for example `FeedItemFilterDialog.newInstance`